firefox-ios/Client/Frontend/Browser/MainMenu/Views/MainMenuViewController.swift

```diff
@@ -376,8 +376,12 @@ class MainMenuViewController: UIViewController,
             return false
         }
 
-        // Don't display CFR for fresh installs
+        // Don't display CFR for fresh installs for users that never saw before the photon main menu
         if InstallType.get() == .fresh {
+            if let photonMainMenuShown = profile.prefs.boolForKey(PrefsKeys.PhotonMainMenuShown),
+               photonMainMenuShown {
+                return viewProvider.shouldPresentContextualHint() ? true : false
```

Comment:
I'm confused here
```swift
                return viewProvider.shouldPresentContextualHint() ? true : false
```
if true return true and if false return false? Why are we doing this?

Reply:
That is true... changed it! Thank you!